concepts/05 UI Components/HtmlEditor/05 Tags and Attributes.md

```diff
@@ -0,0 +1,77 @@
+The following tables list HTML tags and attributes **HtmlEditor** generates.
+
+**HTML Tags**
+
+<table class="dx-table full-width">
+ <tr>
+    <th>Type</th>
+    <th>Tags</th>
+ </tr>
+  <tr>
+    <td>Content sectioning</td>
+    <td>&lt;h1&gt; - &lt;h6&gt;</td>
+ </tr>
+   <tr>
+    <td>Text content</td>
+    <td>&lt;blockquote&gt;, &lt;div&gt;, &lt;ol&gt;, &lt;li&gt;, &lt;ul&gt;, &lt;p&gt;, &lt;pre&gt;</td>
+ </tr>
+   <tr>
+    <td>Inline text</td>
+    <td>&lt;a&gt;, &lt;b&gt;, &lt;br&gt;, &lt;code&gt;, &lt;em&gt;, &lt;i&gt;, &lt;s&gt;, &lt;span&gt;, &lt;strike&gt;, &lt;strong&gt;, &lt;sub&gt;, &lt;sup&gt;, &lt;u&gt;</td>
+ </tr>
+<tr>
+    <td>Images and frames</td>
+    <td>&lt;img&gt;, &lt;iframe&gt;</td>
+ </tr>
+ <tr>
+    <td>Table content</td>
+    <td>&lt;table&gt;,&lt;tbody&gt;,&lt;tr&gt;, &lt;td&gt;</td>
+ </tr>
+</table>
+
+**Attributes**
+
+<table class="dx-table full-width">
```

Comment:
A table with just one column looks a bit weird. How about we make it into a list?